src/test/java/org/apache/commons/collections4/bloomfilter/SparseBloomFilterTest.java

```diff
@@ -33,66 +33,31 @@ protected SparseBloomFilter createEmptyFilter(final Shape shape) {
 
     @Override
     protected SparseBloomFilter createFilter(final Shape shape, final Hasher hasher) {
-        return new SparseBloomFilter(shape, hasher);
```

Comment:
Since these methods just do the same as `createEmptyFilter(Shape)` then `merge(source)` I do not understand why they have to be overridden. The implementation can be in the Abstract test:
```Java
protected BloomFilter createFilter(final Shape shape, final Hasher hasher) {
    BloomFilter bf = createEmptyFilter(shape);
    bf.merge(hasher);
    return bf;
}
```
Is it because the return argument type is incorrect? Then this can be fixed with:
```Java
protected SparseBloomFilter createFilter(final Shape shape, final Hasher hasher) {
    return (SparseBloomFilter) super.createFilter(shape, hasher);
}
```
Doing it this way ensures all filters are using the same create and merge method to populate the new filter.